networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-hosted-zone-infoblox_{context}"]
+= Creating DNS records on an public hosted zone for Infoblox by using Red Hat External DNS Operator
```

Comment:
The term `hosted` can be replaced by `DNS` everywhere cause `hosted` is more of AWS terminology.

Reply:
Ack'ed